app/src/main/res/values/colors.xml

```diff
@@ -6,5 +6,6 @@
     <color name="teal_200">#FFBB86FC</color>
     <color name="teal_700">#FFBB86FC</color>
     <color name="black">#FF000000</color>
+    <color name="black_font">#FF0C111D</color>
```

Comment:
Why could the pre-existing "black" colour not be used?

Reply:
Black is not exactly the same color as the one we currently have as background for the app. I choosed to be coherent with the current background. However, it is easy to change this in the follwing weeks if needed.